schemas/schema.js

```diff
@@ -1064,11 +1065,13 @@ const schema = {
                 {
                   type: "required",
                 },
+              ],
+              fields: [
                 {
-                  type: validatorTypes.URL,
```

Comment:
@nathanbaleeta can you confirm that you remove the URL validator because you expect text input in addition to URLs?

Reply:
If so, we may need to update the text to reflect the fact that we are expecting both URL and text.

Reply:
The API threw an error on adding the above validation hence reason for dropping it. I will update the text to indicate the field captures both URL and text.

Reply:
The API throws an error on adding the above validation for URL when using the `fieldArray` component sometimes. For now will resort to indicating on the label that this field accepts both text and URLs.

Reply:
@nathanbaleeta Can you link to the error, please?

Reply:
Resolved on #21

Reply:
Resolved in 6a6f8f2